backend/routes/usersRoutes.js

```diff
@@ -19,6 +19,13 @@ router.post("/user/register", register);
 router.post("/user/login",login);
 
 
+// Forget password
```

Comment:
When writing code, it is important to use descriptive function names that clearly convey their purpose. In this case, the suggested function names are **sendOTPForForgotPassword** and **verifyOtpForForgotPassword.** and **confirmPassword**